PaymentButtons/src/main/java/com/paypal/android/paymentbuttons/PayPalCreditButton.kt

```diff
@@ -40,10 +40,10 @@ class PayPalCreditButton @JvmOverloads constructor(
             updateShapeDrawableFillColor(field)
         }
 
-    // NEXT_MAJOR_VERSION: - Remove monochrome credit logo
     override val wordmarkDarkLuminanceResId: Int = R.drawable.wordmark_paypal_credit_monochrome
 
-    override val wordmarkLightLuminanceResId: Int = R.drawable.wordmark_paypal_credit_color
+    override val wordmarkLightLuminanceResId: Int
```

Comment:
Is this a breaking change?

Reply:
While no code change is necessary on the merchant's end, the behavior could cause significant disruption to their checkout flow.

Good call: This might have to be updated in the next major version.